Data Cleaing [follow up]

bareshub opened the pull request:
### Description
Follow up data cleaning to support new events, such as `REQUEST_VISIT`, and introduce estimated duration for test and visit events.

### References
- #22

bareshub committed:
fix(data preprocessing): replace ambulance waiting time with ambulance duratione

bareshub committed:
fix(xes): create ArrivalEventStart and ArrivalEventComplete to handle ambulance arrival duration

bareshub committed:
fix(output): update filtered data and log.xes file

bareshub committed:
doc(data preprocessing): perform follow up data cleaning step to add duration and new events to the log

bareshub committed:
feat(xes): create new 'request' events, handle durations and add 'Start' prefix to some events

bareshub committed:
feat(xes): update new up-to-date log

bareshub committed:
feat(xes): update manually cleaned xes log to be used to generate the petri net model

sofiaa21 commented on scripts/s01_data_preprocessing.py:
```diff
@@ -288,8 +337,13 @@ def process_data(input_path: Path, output_path: Path) -> None:
+    )
     df = create_test_department_group(df, TEST_DEPARTMENT_GROUPING)
     df = convert_timestamps(df, TIMESTAMP_COLUMNS)
+    # df = update_ambulance_timestamps(df)
```
forgotten or on purpose?

Beatricia replied:
forgotten, deleted now

sofiaa21 commented on scripts/s02_generate_xes_log.py:
```diff
@@ -202,13 +248,18 @@ def get_unique_from_df(df: pd.DataFrame, key: str):
+        #     event_df,
+        #     "registration_ts_complete"
+        # )
+
```
same comment as above

Beatricia replied:
removed

sofiaa21 reviewed (approved):
nice job!

Beatricia committed:
Remove unused code for ambulance timestamp updates

Removed commented-out code for updating ambulance timestamps and related logic.
